### ISSUE TITLE
Run from command line? 

### ISSUE THREAD

**Issue by SpencerRiddering** [tenninebt/vscode-koverage#51]
I like the way this extension presents coverage data, and I'd like to integrate it into my GitLab automated build pipeline. 

- Is there a way to generate this report as a standalone web page? 
- Can the report generation be run from the command line?

**Comment by AlefCS** [comment]
@SpencerRiddering I don't think this extension is able to generate a report. I think it only reads from a report and show the coverage info in VSCode.

**Comment by tenninebt** [comment]
There are already existing tools to convert an lcov (or other formats) to HTML. This is out of scope for this extension, sorry.
